spec/models/manageiq/providers/embedded_ansible/automation_manager/playbook_spec.rb

```diff
@@ -1,7 +1,7 @@
 describe ManageIQ::Providers::EmbeddedAnsible::AutomationManager::Playbook do
   let(:manager) { FactoryGirl.create(:embedded_automation_manager_ansible) }
-  let(:auth_one) { FactoryGirl.create(:authentication, :manager_ref => 6) }
-  let(:auth_two) { FactoryGirl.create(:authentication, :manager_ref => 8) }
+  let(:auth_one) { FactoryGirl.create(:embedded_ansible_credential, :manager_ref => 6) }
+  let(:auth_two) { FactoryGirl.create(:embedded_ansible_credential, :manager_ref => 8) }
```

Comment:
The previous code was setting `manager_ref` as an integer, but it be a string to match the column type.